### ISSUE TITLE
TypeError: this.handleUnhandledRejects is not a function

### ISSUE THREAD

**Issue by FredyC** [rollbar/rollbar.js#266]
Kinda a critical error :) Typo in a function name [here](https://github.com/rollbar/rollbar.js/blob/universal/src/server/rollbar.js#L29) for a server side run.

Besides this, I am very confused. [Official docs](https://rollbar.com/docs/notifier/node_rollbar/) are using `node_rollbar` module approach and yet by installing `rollbar` module I am getting 2.0.0-alpha.5 of the module from this repo which is rather different to use. On top of that, it's very hard to actually find relevant docs as `master` branch is for a client side use only.

It's very unfortunate for a newcomer like this and takes quite a lot of time to figure what's wrong actually.

**Comment by coryvirok** [comment]
Hi Fredy,

Thanks for filing this. I'm taking a look now. As for docs, the v2 alpha release hasn't been merged back into master yet as we are still doing some testing. The new version is a universal JS SDK for Rollbar whereas before there was node_rollbar and rollbar.js. The docs for the alpha version can be found [here](https://github.com/rollbar/rollbar.js/blob/universal/README.md).

**Comment by FredyC** [comment]
Yes, I found docs after like an hour of digging through... I must say it's not really good idea to publish with `latest` tag something that isn't merged into `master`. Makes things super confusing. I think it would really help if you add a message to README in `master` with information that `latest` NPM module docs are found in the branch.

Also, some deprecation notice to `node_rollbar` and pointing to this repo would be very useful.

**Comment by rokob** [comment]
This is fixed via the referenced merged PR. Sorry about the confusion with latest being on a branch. I will close this once the other issues you have addressed are taken care of.

**Comment by rokob** [comment]
I was unaware that npm publish automatically set the latest pointer, sorry about that. I pushed a new version of node_rollbar just to move the latest pointer for the `rollbar` package back to master of that repo. I also added a beta tag which points to the package made from the universal branch of this repo. I added a deprecation note to the CHANGELOG file on node_rollbar, and I will add something more prominent to the README once we actually fully release the universal rollbar.js repo in this repo.
